src/components/Form.js

```diff
@@ -1,35 +1,74 @@
 import React from 'react'
 import styled from 'styled-components/macro'
 import { NavLink, useHistory } from 'react-router-dom'
+import { useForm } from 'react-hook-form'
+import { v4 as uuidv4 } from 'uuid'
 
 export default function Form({ onSubmit }) {
   const history = useHistory()
+  const { register, handleSubmit, errors } = useForm()
+  const onFormSubmit = (newReview) => {
+    onSubmit({ ...newReview, id: uuidv4() })
+    history.push('/')
+  }
   return (
-    <StyledForm onSubmit={handleSubmit}>
+    <StyledForm onSubmit={handleSubmit(onFormSubmit)}>
       <label>
         Title
-        <Input autoFocus name="title" />
+        <Input autoFocus name="title" ref={register({ required: true })} />
```

Comment:
Your naming of components is inconsistent. Since `Input` is only a styled component, you could name it `StyledInput` just like your component `StyledForm`.
Same goes for `ButtonGroup`.

Reply:
Good point, I will change it to a more consistent naming!